experiments/benchmarks/bucket.jl

```diff
@@ -143,21 +142,29 @@ function setup_prob(t0, tf, Δt)
     return prob, cb
 end
 
-function setup_and_solve_problem()
+function setup_and_solve_problem(; greet = false)
     # We profile the setup phase as well here. This is not intended, but it is the easiest
     # to set up for both CPU/GPU at the same time
     t0 = 0.0
     tf = 7 * 86400
     Δt = 3600.0
-    prob, cb = setup_prob(t0, tf, Δt)
+    nelements = (100, 10)
+    if greet
+        @info "Run: Bucket with temporal map"
+        @info "Resolution: $nelements"
+        @info "Timestep: $Δt"
+        @info "Duration: $tf"
```

Comment:
It might be better to use `$(tf - t0)` so we aren't assuming `t0 = 0`, and it would also be nice to include that the units for `dt` and `tf` are seconds

Reply:
Added, thanks!